lib/liftoff/project.rb

```diff
@@ -63,6 +64,18 @@ def set_company_name(company)
       xcode_project.root_object.attributes['ORGANIZATIONNAME'] = company
     end
 
+    def create_build_configurations(build_configurations)
+      builder = BuildConfigurationBuilder.new(xcode_project)
+      build_configurations ||= []
+
+      build_configurations.each do |configuration|
```

Comment:
This was my first thought, but then I thought that it'd be better if `BuildConfigurationBuilder` didn't have to deal with any hashes (to avoid an informal contract about the keys). Does that make sense?